lib/services/project-changes-service.ts

```diff
@@ -227,10 +228,15 @@ export class ProjectChangesService implements IProjectChangesService {
 
 			let fileStats = this.$fs.getFsStats(filePath);
 
-			let changed = fileStats.mtime.getTime() >= this._outputProjectMtime || fileStats.ctime.getTime() >= this._outputProjectCTime;
-			if (!changed) {
-				let lFileStats = this.$fs.getLsStats(filePath);
-				changed = lFileStats.mtime.getTime() >= this._outputProjectMtime || lFileStats.ctime.getTime() >= this._outputProjectCTime;
+			let changed =  fileStats.mtime.getTime() > this._outputProjectMtime ||
+				fileStats.ctime.getTime() >= this._outputProjectCTime;
+
+			if (this.$hostInfo.isDarwin && !fileStats.isDirectory() && !changed) {
```

Comment:
I agree with you, but I don't know a trivial way of finding out what operating system is the code running on. That's why I check for darwin, because that's where the problem presents itself.